neo/Helper.cs

```diff
@@ -181,6 +183,11 @@ public static uint ToTimestamp(this DateTime time)
             return (uint)(time.ToUniversalTime() - unixEpoch).TotalSeconds;
         }
 
+        public static uint ToTimestampMS(this DateTime time)
+        {
+            return (uint)(time.ToUniversalTime() - unixEpoch).TotalMilliseconds;
```

Comment:
Brother, perhaps this only has seconds precision, not milliseconds. unixEpoch is typically a 'seconds' stuff.